docs/_jul_milestone_.md

```diff
@@ -0,0 +1,23 @@
+# Monthly Milestone - July 2020
+
+## Overview
+
+- Docs issue takedown!
+
+## Goals
+
+- [ ] Docs improvements
+
+  _Please add your suggestions here. Also see
+  https://github.com/strongloop/loopback-next/issues?q=is%3Aissue+is%3Aopen+label%3ADocs+label%3A2020Q3_
+
```

Comment:
Add missing LoopBack 4 Types docs https://github.com/strongloop/loopback-next/issues/5063
Docs: how to configure a custom table/collection name different from model class name #2197 ( I think I've done this for those main connectors but it'd be better to check it thoroughly)
Maybe a good stretch goal: couchDB connector docs #5267

Reply:
Regression: MongoError: unknown operator: $$eq  https://github.com/strongloop/loopback-connector-mongodb/issues/532